jsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+    "compilerOptions": {
+        "baseUrl": ".",
+        "paths": {
+            "@/components/*": [
+                "components/*"
+            ]
```

Comment:
We can add path aliases for config and other folders later on. This is customizable.

Reply:
Great idea, thank you